installer/conf/omsagent.d/npmd.conf

```diff
@@ -3,7 +3,7 @@
     tag oms.npmd
     location_unix_endpoint  "/var/opt/microsoft/omsagent/run/npmdagent.sock"
     location_control_data   "/etc/opt/microsoft/omsagent/conf/npmd_agent_config.xml"
-    location_agent_binary   "/opt/microsoft/omsagent/npmd_agent"
+    location_agent_binary   "/opt/microsoft/omsagent/plugin/npmd_agent"
```

Comment:
Matching the version of npmd.conf present in DSC.
[Link to npmd.conf binary location line](https://github.com/Microsoft/PowerShell-DSC-for-Linux/blob/master/Providers/Modules/NPM/Plugin/conf/npmd.conf#L6)